package.json

```diff
@@ -22,7 +22,7 @@
     "lint:hbs": "ember-template-lint app",
     "lint:js": "eslint . --cache",
     "precompress": "node ./script/precompress-assets.mjs",
-    "prettier": "prettier --write package.json '**/*.js' '**/*.mjs'",
+    "prettier": "prettier --write package.json '**/*.js' '**/*.mjs' '**/*.md'",
```

Comment:
keep in mind though that on CI we only check prettier stuff via ESLint, so there is a risk of the files diverging again in the future